pipelines/iot_analytics/scripts/00_set_environment.sh

```diff
@@ -0,0 +1,16 @@
+export PROJECT_ID=learnings-421714
```

Comment:
Please remove this file from the PR. This file is generated by the Terraform code (see the main.tf file, last part of the file). If not all variables are set in the Terraform code, please include them in the Terraform code.